Jenkinsfile

```diff
@@ -26,42 +27,45 @@ node {
     // JWT key credentials.
     // -------------------------------------------------------------------------
 
-    withCredentials([file(credentialsId: SERVER_KEY_CREDENTIALS_ID, variable: 'server_key_file')]) {
-        // -------------------------------------------------------------------------
-        // Authenticate to Salesforce using the server key.
-        // -------------------------------------------------------------------------
-
-        stage('Authorize to Salesforce') {
-            rc = command "${toolbelt}/sfdx force:auth:jwt:grant --instanceurl https://test.salesforce.com --clientid ${SF_CONSUMER_KEY} --jwtkeyfile ${server_key_file} --username ${SF_USERNAME} --setalias UAT"
-            if (rc != 0) {
-                error 'Salesforce org authorization failed.'
-            }
-        }
-
-
-        // -------------------------------------------------------------------------
-        // Deploy metadata and execute unit tests.
-        // -------------------------------------------------------------------------
-
-        stage('Deploy and Run Tests') {
-            rc = command "${toolbelt}/sfdx force:mdapi:deploy --wait 10 --deploydir ${DEPLOYDIR} --targetusername UAT --testlevel ${TEST_LEVEL}"
-            if (rc != 0) {
-                error 'Salesforce deploy and test run failed.'
-            }
-        }
-
-
-        // -------------------------------------------------------------------------
-        // Example shows how to run a check-only deploy.
-        // -------------------------------------------------------------------------
-
-        //stage('Check Only Deploy') {
-        //    rc = command "${toolbelt}/sfdx force:mdapi:deploy --checkonly --wait 10 --deploydir ${DEPLOYDIR} --targetusername UAT --testlevel ${TEST_LEVEL}"
-        //    if (rc != 0) {
-        //        error 'Salesforce deploy failed.'
-        //    }
-        //}
-    }
+ 	withEnv(["HOME=${env.WORKSPACE}"]) {	
+	
+	    withCredentials([file(credentialsId: SERVER_KEY_CREDENTIALS_ID, variable: 'server_key_file')]) {
+		// -------------------------------------------------------------------------
+		// Authenticate to Salesforce using the server key.
+		// -------------------------------------------------------------------------
+
+		stage('Authorize to Salesforce') {
+			rc = command "${toolbelt}/sfdx force:auth:jwt:grant --instanceurl ${SF_INSTANCE_URL} --clientid ${SF_CONSUMER_KEY} --jwtkeyfile ${server_key_file} --username ${SF_USERNAME} --setalias UAT"
```

Comment:
Release hardcoded instance URL with var `SF_INSTANCE_URL`

Reply:
@peternhale this file appears to have the env var already set: --instanceurl ${SF_INSTANCE_URL}. I updated the sfdx-jenkins-package file accordingly, but don't see an edit to make here in sfdx-jenkins-org.